### PR TITLE
OCPBUGS-16649: Accelerated startup cleanup

### PR DESCRIPTION
**Pull request by lack**

### CODE REVIEW

File: ztp/extra-manifests-builder/04-accelerated-container-startup/accelerated-container-startup.sh [hunk 1]
```diff
@@ -144,31 +144,33 @@ steadystate() {
...
       lastSystemdCpuset="$(currentAffinity 1)"
       lastDesiredCpuset="$desiredCpuset"
     fi
 
     # Detect steady-state pod count
-    ccount=$(crictl ps | wc -l)
-    if steadystate $lastCcount $ccount; then
+    ccount=$(crictl ps 2>/dev/null | wc -l)
+    if [[ $ccount -gt 0 ]] && steadystate "$lastCcount" "$ccount"; then
       ((steadyStateTime += s))
```
Comment (lack) [inline ztp/extra-manifests-builder/04-accelerated-container-startup/accelerated-container-startup.sh:174]: This is the only function change; Hiding the stderr output from crictl and ignoring the result if the total number of running containers is 0

File: ztp/extra-manifests-builder/04-accelerated-container-startup/accelerated-container-startup.sh [hunk 2]
```diff
@@ -144,31 +144,33 @@ steadystate() {
...
-  local lastSystemdCpuset="$(currentAffinity 1)"
-  local lastDesiredCpuset="$(unrestrictedCpuset)"
+  local lastSystemdCpuset lastDesiredCpuset
+  lastSystemdCpuset=$(currentAffinity 1)
+  lastDesiredCpuset=$(unrestrictedCpuset)
   local t=0 s=10
   local lastCcount=0 ccount=0 steadyStateTime=0
   while [[ $t -lt $MAXIMUM_WAIT_TIME ]]; do
     sleep $s
     ((t += s))
```
Comment (donpenney) [inline ztp/extra-manifests-builder/04-accelerated-container-startup/accelerated-container-startup.sh:159]: Just an fyi... bashate is also flagging some potential issues. Mostly indentation and line length (I ignore the line lengths when I add bashate and shellcheck as linters in repos), but has flagged this arithmetic expression style in a few spots:

accelerated-container-startup.sh:80:1: E043 Arithmetic compound has inconsistent return semantics
accelerated-container-startup.sh:82:1: E043 Arithmetic compound has inconsistent return semantics
accelerated-container-startup.sh:159:1: E043 Arithmetic compound has inconsistent return semantics
accelerated-container-startup.sh:174:1: E043 Arithmetic compound has inconsistent return semantics

Comment (lack) [reply ztp/extra-manifests-builder/04-accelerated-container-startup/accelerated-container-startup.sh:159]: I don't know bashate - Sounds like something I wouldn't hate.

**Pull request merged**